<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1643926466034' stop='1643926491703'>
    <name>Contact</name>
    <title>Contact</title>
    <test-cases>
        <test-case start='1643926466035' status='failed' stop='1643926491703'>
            <name>Fill all the input fields, submit form and assert success message</name>
            <title>Fill all the input fields, submit form and assert success message</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-97.0.4692.99'/>
            </parameters>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='66cf3151-794a-42c6-84bf-884cbe5ecc3f-attachment.png' type='image/png' size='1014156'/>
            </attachments>
            <failure>
                <message>Expect $(`[role="alert"]`) to have text containing
                
                Expected: "Thank you for contacting us! We will be in touch with you shortly"
                Received: "Thanks for contacting us! We will be in touch with you shortly"</message>
                <stack-trace>Error: Expect $(`[role="alert"]`) to have text containing
                
                Expected: "Thank you for contacting us! We will be in touch with you shortly"
                Received: "Thanks for contacting us! We will be in touch with you shortly"
                    at Context.&lt;anonymous> (/Users/holleyvoegtle/Desktop/Projects for CCCareers/wdio-course/test/specs/contact.js:22:36)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>